openedx/core/djangoapps/coursegraph/management/commands/tests/test_dump_to_neo4j.py

```diff
@@ -228,7 +248,54 @@ def test_serialize_course(self):
         """
         nodes, relationships = self.mss.serialize_course(self.course.id)
         self.assertEqual(len(nodes), 9)
-        self.assertEqual(len(relationships), 7)
+        # the course has 7 "PARENT_OF" relationships and 3 "PRECEDES"
+        self.assertEqual(len(relationships), 10)
+
+    def assertBlockPairIsRelationship(self, xblock1, xblock2, relationships, relationship_type):
+        """
+        Helper assertion that a pair of xblocks have a certain kind of
+        relationship with one another.
+        """
+        relationship_pairs = [
+            tuple([node["location"] for node in rel.nodes()])
+            for rel in relationships if rel.type() == relationship_type
+        ]
+        location_pair = (unicode(xblock1.location), unicode(xblock2.location))
+        self.assertIn(location_pair, relationship_pairs)
+
+    def assertBlockPairIsNotRelationship(self, xblock1, xblock2, relationships, relationship_type):
+        """
+        The opposite of `assertBlockPairIsRelationship`: asserts that a pair
+        of xblocks do NOT have a certain kind of relationship.
+        """
+        relationship_pairs = [
```

Comment:
@awaisdar001 sure. I was trying to decide if it would be overkill to have that extra layer of abstraction, but I could totally do it